comictaggerlib/taggerwindow.py

```diff
@@ -1761,10 +1777,13 @@ def metadata_save() -> bool:
         ii = IssueIdentifier(ca, self.config[0], self.current_talker())
 
         # read in metadata, and parse file name if not there
+        # TODO should this follow the same as CLI: filename (-f), read styles (-t), command line (-m)
+        # Duplicated in autotagmatchwindow and renamewindow
+        md = GenericMetadata()
```

Comment:
Will users expect consistency between CLI and GUI? CLI users are more likely to read the CLI options, of course. When `overlyMode` comes into play too... I think `filename` then `styles` and `-m` makes sense unless an option is disabled/empty.